Rust/Rust.sublime-syntax

```diff
@@ -1393,6 +1395,35 @@ contexts:
           scope: punctuation.definition.string.end.rust
           pop: true
 
+  c-string:
+    - match: '(c)(")'
+      captures:
+        1: storage.type.string.rust
+        2: punctuation.definition.string.begin.rust
+      push:
+        - meta_include_prototype: false
+        - meta_scope: string.quoted.double.rust
+        - match: '"'
+          scope: punctuation.definition.string.end.rust
+          pop: true
+        - match: '{{escaped_byte}}'
+          scope: constant.character.escape.rust
+        - match: '(\\)$'
```

Comment:
Why capture group if you're not going to use it?

Reply:
Not sure I'd call a tiny oversight a "tradition". And now is the perfect time to improve things like that - some engines which use `sublime-syntax` files can get a small performance boost from not using unnecessary capture groups.

Reply:
If some matches end up being duplicated, it might make sense to factor them out into a new context which can be shared/included in the relevant places. It might be the right time to do this for the line continuation and escaped byte matches.